client/components/status/status.js

```diff
@@ -211,23 +231,25 @@ export default class Status extends React.Component {
       captivePortalLogoutForm,
       isAuthenticated,
     } = this.props;
-    const {content, links, buttons} = statusPage;
-    const {username, password, sessions} = this.state;
-    const contentArr = getText(content, language).split("\n");
+    const {links, buttons} = statusPage;
+    const {username, password, sessions, user_info} = this.state;
     return (
       <>
         <div className="container content" id="status">
           <div className="inner">
             <div className="main-column">
-              {contentArr.map(text => {
-                if (text !== "")
-                  return (
-                    <p key={text}>
-                      {text}
-                    </p>
-                  );
-                return null;
-              })}
+              <table>
+                <tbody>
+                  {Object.entries(user_info).map(([key, value]) => {
+                      return (
+                        <tr key={key}>
+                          <td>{key}:</td>
+                          <td>{value}</td>
+                        </tr>
+                      );
```

Comment:
Please define the fields one by one and get the translatable text in a way that is consistent with the rest of the application (create one config key for each word).